classes/main.php

```diff
@@ -135,16 +135,12 @@ public function locate_image( $tpl ) {
 		// Rework the tpl
 		$tpl = str_replace( '_', '-', $tpl );
 
-		if ( is_file( sprintf( '%s/%s.png', $path, $tpl ) ) ) {
-			return sprintf( '%s/%s.png', $path, $tpl );
-		}
-
-		if ( file_exists( sprintf( '%s/%s/%s.png', get_stylesheet_directory(), $path, $tpl ) ) ) {
-			return sprintf( '%s/%s/%s.png', get_stylesheet_directory_uri(), $path, $tpl );
-		}
-
-		if ( file_exists( sprintf( '%s/%s/%s.png', get_template_directory(), $path, $tpl ) ) ) {
-			return sprintf( '%s/%s/%s.png', get_template_directory_uri(), $path, $tpl );
+		foreach( ['jpg', 'jpeg', 'png', 'gif'] as $extension ) {
+			$image = sprintf('%s/%s.%s', $path, $tpl, $extension);
+			if( !is_file($image) && !is_file( get_theme_file_path($image) ) ) {
```

Comment:
Some review here :
- needs to be refactored with coding standards
- if $image exists, it is never returned
- get_theme_file_path($image) ) should be stocked into a var to avoid to get it a second time
- moreover, get_theme_file_path() don't really check if files are present, so the default case will not anymore trigger

Reply:
- I always try to stick to coding standards when there are some but this repo doesn't provide any. No phpcs.xml or other type of CS file and nothing is documented about CS in the contributing file
- My mistake, I made this quick PR to fit my needs because my images are placed in my theme folder, I didn't check other cases. Anyway, the actual code [won't return an image placed in the plugin folder either](https://github.com/BeAPI/bea-beautiful-flexible/blob/master/classes/main.php#L138-L140), unless you place them in `classes/assets/bea-beautiful-flexible` which is very unlikely. It's also not a good practice to use the plugin folder to store any user/developer specific files as it can be easily removed. Would you consider removing that location?
- `get_theme_file_path` is only used once. 
- That's why `is_file` does. `get_theme_file_path` is a core function to avoid handling [child theme case](https://github.com/BeAPI/bea-beautiful-flexible/blob/master/classes/main.php#L142-L148) yourself.